README.md

```diff
@@ -83,6 +84,50 @@ npm run bundle -- --importjs tools/importer/import.js
 
 The `import.js` file you provide will be bundled with any locally referenced scripts and saved to a new file named `import.bundle.js`.
 
+### Assistant
+
+Run AI-enabled commands to assist with your import script development.
+
+Set your environment variables (either in your shell profile or in a `.env` file):
+
+```
+export IMS_AUTH_CODE=<your ims auth code>
+export IMS_CLIENT_SECRET=<your ims client secret>
+```
```

Comment:
Can be removed from the README, looking ahead to having a service that will manage this for us.